crates/daphne/src/vdaf/prio2.rs

```diff
@@ -71,7 +77,10 @@ pub(crate) fn prio2_prep_init(
     <()>::get_decoded_with_param(&vdaf, public_share_data)?;
     let input_share: Share<FieldPrio2, 32> =
         Share::get_decoded_with_param(&(&vdaf, agg_id), input_share_data)?;
-    let (state, share) = vdaf.prepare_init(verify_key, agg_id, &(), nonce, &(), &input_share)?;
+    let binding = [CTX_STRING, &task_id.0].concat();
+    let ctx = binding.as_slice();
+    let (state, share) =
+        vdaf.prepare_init(verify_key, ctx, agg_id, &(), nonce, &(), &input_share)?;
```

Comment:
Hmmm ... there are no functional changes to Prio2 between `prio_draft09` and `prio`. The implementation will simply ignore the context string, so there's not much point in constructing it here.

Stepping back, I don't think we need two version so the `prio2` module, as there is only one version of this algorithm we need to support (either in DAP-09 or DAP-13).

FYI, in case this wasn't clear: Prio2 is not standardized, but is provided by libprio. It hasn't changed at all since DAP-02.